terraform/README.md

```diff
@@ -0,0 +1,83 @@
+# Airbyte Terraform
+
+## Connector Development Infrastructure
+We use Terraform to manage any persistent infrastructure used for developing or testing connectors.
+
+Directory structure is roughly as follows:
+
+    ├── aws
+    │   ├── demo
+    │   │   ├── core
+    │   │   └── lb
+    │   ├── shared
```

Comment:
Should we move all connector-related infra to `airbyte-integrations/infra`? Probably better to scope it that way but if it causes disruption then we don't need to

Reply:
Yeah, I agree.  Moved.